addons/sourcemod/scripting/nominations_extended.sp

```diff
@@ -189,6 +206,18 @@ public void OnSettingsChanged(Handle convar, const char[] oldValue, const char[]
 	}
 }
 
+public void OnMapStart()

```

Comment:
This should happen only if KZTimer or GOKZ are found, and if globals are enabled

Reply:
Follow-up: In its current state, this will trigger on every server that runs this plugin, KZ or not, which is unnecessary. This will also run on non-global servers as well. If you would like it to run on non-global servers, perhaps add a convar switch, but generally speaking if it is non-global there is a reason for it.